test/ngAria/ariaSpec.js

```diff
@@ -750,6 +750,18 @@ describe('$aria', function() {
     });
   });
 
+  describe('actions when bindRoleForClick is set to false', function() {
+      beforeEach(configAriaProvider({
+          bindRoleForClick: false
+      }));
+      beforeEach(injectScopeAndCompiler);
+
+      it('should not add a button role', function() {
+        compileElement('<radio-group ng-click="something"></radio-group>');
+        expect(element.attr('role')).toBeUndefined();
+      });
+  });
```

Comment:
Inside the function you are using 4 spaces (most of the time). It should be 2 spaces (according to Angular's "styleguide".
(I don't think it's a big deal, but I have a thing about whitespace inconsistency :P)